hack/debug.sh

```diff
@@ -468,6 +502,8 @@ for systemd_dir in /etc/systemd/system /usr/lib/systemd/system; do
     for name in openshift origin atomic-openshift; do
 	if [ -f $systemd_dir/$name-master.service ]; then
 	    aos_master_service=$name-master.service
+	    aos_master_controllers_service=$name-master-controllers.service
+	    aos_master_api_service=$name-master-api.service
```

Comment:
Will `atomic-openshift-master.service` always exist on HA systems, or should we do `if [ -f ...` tests for each file separately?